docs/reference/organizing-code/organizing-code.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Organizing Code"
+sidebar_label: "Organizing Code"
+sidebar_position: 1
+keywords:
+  - modules
+  - namespace
+draft: false
+---
+
+A high level description of how code is organized in Darklang.
```

Comment:
This is a good page to have. I would suggest leaving this until we figure out how code gets put in the package manager, as we don't necessarily expect to have modules in the same way as we do now.